wiki/BanchoBot/de.md

```diff
@@ -1,25 +1,158 @@
+
+Wenn ein normaler Benutzer einen Befehl in einem öffentlichen Chat-Kanal sendet, sehen andere Benutzer ihn nicht, und die Antwort wird in einer privaten Nachricht mit BanchoBot angezeigt. Benutzer können auch den Befehl `/bb` im Spiel-Client verwenden, um automatisch einen Tab mit BanchoBot zu öffnen und den Befehl sofort zu senden.
+
+Eine Liste aller BanchoBot-Befehle befindet sich unten:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` zeigt eine Liste aller verfügbaren BanchoBot-Befehle an. Ein Beispiel für das Senden dieses Befehls wird unten gezeigt:
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+```
+
+<!--note for editors: the code block above reflects the exact response from banchobot -->
+
+*Tipp: Der Befehl `!request` wird von BanchoBot nicht mehr unterstützt.
+
+### Roll
+
+```
+!roll <Argument>/<Zahl>
+```
+
+`!roll` zieht eine Zufallszahl zwischen 1 und der ausgewählten Zahl. Wenn die Zahl nicht angegeben wird oder ein Argument angegeben wird, wird die maximale Zahl auf 100 gesetzt. Beispiele für das Senden dieses Befehls sind unten aufgeführt:
+
+```
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
+```
+
+```
+13:01 pippi: !roll probability of failure
+13:01 BanchoBot: pippi rolls 75 point(s)
+```
+
+### Stats
+
+```
+!stats <Benutzername>
+```
+
+`!stats` zeigt die Spielstatistiken des eingegebenen Benutzers und seinen aktuellen Status an. Die Ausgabe hängt von dem [Spielmodus](/wiki/Game_mode) ab, den der eingegebene Benutzer zuletzt gespielt hat, obwohl BanchoBot nicht anzeigt, aus welchem Spielmodus die Statistiken stammen. Wenn BanchoBot die Statistiken eines Benutzers anzeigen soll, der noch nie osu! gespielt hat, antwortet er mit "Benutzer nicht gefunden", auch wenn der Benutzer existiert. Ein Beispiel für das Senden dieses Befehls wird unten gezeigt:
+
+```
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
+```
+
+Es gibt insgesamt 7 Stati, die mit dem Befehl `!stats` angezeigt werden können: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying und Playing. Ein Beispiel für das Senden dieses Befehls für einen Benutzer mit einem verfügbaren Status ist unten dargestellt:
```

Comment:
```suggestion
Es gibt insgesamt 7 Status, die mit dem Befehl `!stats` angezeigt werden können: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying und Playing. Ein Beispiel für das Senden dieses Befehls für einen Benutzer mit einem verfügbaren Status ist unten dargestellt:
```
[Duden suggests `Status` regardless of the grammatical form](https://www.duden.de/deklination/substantive/Status). Might need a moment myself to get used to this.

Reply:
interesting